api/mod/src/settings.py

```diff
@@ -19,6 +19,7 @@ class Settings(BaseSettings):
     """
 
     data_folder: Path = Path.home().joinpath(".annoto")
+    database_url: str = f"sqlite:///{Path.home().joinpath('.annoto').joinpath('db.db')}"
```

Comment:
Interesting file extension `.db` :thinking: I mean, it obviously works like this, but maybe we should consider using something more descriptive (and I think also more common?) like `.sqlite3`?